### PR TITLE
feat: multiple actions appear side-by-side

### PR DESCRIPTION
**Pull request by pr3tori4n**
**Related Issue:** #753 

## Summary

What Adelheide is asking for is already possible by passing in more than one action using the same slot. The only thing missing was displaying them properly. Today they appear one atop of the other. With this small CSS change, they will appear side by side.

QUESTION: Matt brought up making the slot name plural `secondary-action(s)`. I don't mind the update. All slots by nature can accept multiple slotted elements - so this may speak to our slot naming conventions a bit.

### CODE REVIEW

File: src/demos/value-list/doubleAction.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```
Comment (jcfranco) [inline src/demos/value-list/doubleAction.html]: This filename's casing is inconsistent with existing demos.

**Comment by asangma** [comment]
👀

**Comment by pr3tori4n** [comment]
@asangma yea can you add styles for the other cases?

File: src/demos/value-list/doubleAction.html (hunk 1 above)
Comment (jcfranco) [reply src/demos/value-list/doubleAction.html]: @pr3tori4n This was not resolved before merging. Can you fix it?

**Pull request merged**